<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGraphDialog</class>
 <widget class="QDialog" name="LoadGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loadFromURIButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load From URI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFromFileButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load From File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphURIEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="graphURILabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph URI:</string>
   </property>
  </widget>
  <widget class="QLabel" name="OrLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
